telegram/bot.py

```diff
@@ -132,6 +106,27 @@ def __init__(self, token, base_url=None, base_file_url=None, request=None, priva
                                                                   password=private_key_password,
                                                                   backend=default_backend())
 
+    def _message(self, url, data, **kwargs):
+        if kwargs.get('reply_to_message_id'):
+            data['reply_to_message_id'] = kwargs.get('reply_to_message_id')
+
+        if kwargs.get('disable_notification'):
+            data['disable_notification'] = kwargs.get('disable_notification')
+
+        if kwargs.get('reply_markup'):
+            reply_markup = kwargs.get('reply_markup')
+            if isinstance(reply_markup, ReplyMarkup):
+                data['reply_markup'] = reply_markup.to_json()
+            else:
+                data['reply_markup'] = reply_markup
+
+        result = self._request.post(url, data, timeout=kwargs.get('timeout'))
+
+        if result is True:
```

Comment:
Just to be sure. You checked it's always either `True` or a `Message`?

Reply:
I mean... that exact code worked before... so I would assume?

Reply:
Alright, just checked, and all the methods that we use the _message helper in, do indeed return either True or a Message.